core/client/app/components/gh-profile-image.js

```diff
@@ -18,22 +18,22 @@ export default Ember.Component.extend({
     email: '',
     size: 90,
     hasUploadedImage: false,
+    imageBackground: '',
 
     ghostPaths: Ember.inject.service('ghost-paths'),
-    hasEmail: Ember.computed.notEmpty('email'),
 
     defaultImage: Ember.computed('ghostPaths', function () {
         var url = this.get('ghostPaths.url').asset('/shared/img/user-image.png');
         return `background-image: url(${url})`.htmlSafe();
     }),
 
-    imageBackground: Ember.computed('email', 'size', function () {
+    applyImageBackground: Ember.observer('email', 'size', function () {
```

Comment:
Does this need to be an observer? At a glance I think it should be ok as a computed property.